app/code/Magento/Elasticsearch/Elasticsearch5/SearchAdapter/Query/Builder.php

```diff
@@ -22,6 +22,8 @@
  */
 class Builder
 {
+    private const ELASTIC_INT_MAX = 2147483647;
```

Comment:
```suggestion
    private const ELASTIC_MAX_INT = 2147483647;
```

Reply:
Agree that max_int sounds better but there is defined PHP_INT_MAX constant. I think it is better to keep the same format here.

Reply:
Ok, that make sense